features/networking/operator.feature

```diff
@@ -186,3 +186,57 @@ Feature: Operator related networking scenarios
     And the output should not contain "NotReady"
     Given the status of condition "Progressing" for network operator is :False
     """
+  # @author anusaxen@redhat.com
+  # @case_id OCP-24918
+  @admin
+  Scenario: Service should not get unidle when config flag is disabled under CNO
+  Given I have a project
+  When I run the :create client command with:
+    | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/networking/list_for_pods.json |
+  Then the step should succeed
+  And a pod becomes ready with labels:
+      | name=test-pods |
+  Given I use the "test-service" service
+  And evaluation of `service.ip(user: user)` is stored in the :service_ip clipboard
+  And I wait for the "test-service" service to become ready
```

Comment:
Should "And I wait for the "test-service" service to become ready", then
"And evaluation of `service.ip(user: user)` is stored in the :service_ip clipboard"?

Reply:
We actually don't need to check service become ready step. Removed it for now. As if svc is not ready, it won't get IP assigned and it will eventually fail at evaluation step